IrisSoftware/main.py

```diff
@@ -50,28 +51,42 @@ def detectBlink(self, eyeCoords, blinkDuration) -> any:
         pass
 
     def moveMouse(self, screenX, screenY):
-        '''Move the mouse to the given screen coordinates, moving smoothly over multiple frames'''
+        """Move the mouse to the given screen coordinates, moving smoothly over multiple frames"""
         if self.state.lastCursorPos is None:
             pyautogui.moveTo(screenX, screenY)
             self.state.lastCursorPos = (screenX, screenY)
         else:
             # Smooth out the mouse movement to minimize jitter
-            x = self.state.lastCursorPos[0] + (screenX - self.state.lastCursorPos[0]) * 0.1
-            y = self.state.lastCursorPos[1] + (screenY - self.state.lastCursorPos[1]) * 0.1
+            x = (
+                self.state.lastCursorPos[0]
+                + (screenX - self.state.lastCursorPos[0]) * 0.1
+            )
+            y = (
+                self.state.lastCursorPos[1]
+                + (screenY - self.state.lastCursorPos[1]) * 0.1
+            )
```

Comment:
This expression spacing is kind of awkward, can we leave it as-is?

Reply:
This is the result of our auto-formatter, `black`, in action. With my editor setup, the files that I edit and then save are automatically formatted. If we were to change this back, it would defeat the purpose of the formatted (having a consistent format for our code) and it would make it so that I can't run the formatted when I save `main.py`. So I'd rather leave it like this and leave the formatting to `black` 😄